docusaurus.config.js

```diff
@@ -62,12 +58,13 @@ const config = {
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
       // Replace with your project's social card
-      image: 'img/docusaurus-social-card.jpg',
+      image: 'img/logo-emblem.svg',
       navbar: {
         logo: {
           alt: 'Responsive Logo',
           src: 'img/logo.svg',
           srcDark: 'img/logo-dark.svg',
+          href: 'https://responsive.dev'
```

Comment:
It's been driving me a bit crazy that once you navigate to the docs site, the logo in the top left corner starts pointing you to `docs.responsive.dev` rather than `responsive.dev` like it does everywhere else. It's like a one-way ticket to getting trapped on the docs sub-site